searx/static/themes/simple/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "devDependencies": {
-    "eslint": "^8.0.1",
+    "eslint": "^8.11.0",
```

Comment:
Does it make sense to upgrade eslint in the root also:

https://github.com/searxng/searxng/blob/6ccd557650a0892a3c59ac157a781f9a12beb56e/package.json#L3

Reply:
Actually package-lock.json should be commited, so dependabot can work.

Reply:
My idea:

1. drop oscar
2. merge node env from simple theme with the one in root `.`
3. commit `package-lock.json`

Reply:
Does this block this PR?

Reply:
No, but while we are, we should also update 

https://github.com/searxng/searxng/blob/6ccd557650a0892a3c59ac157a781f9a12beb56e/package.json#L3